module-sys/PhoneModes/Subject.cpp

```diff
@@ -80,6 +81,6 @@ namespace sys::phone_modes
 
     bool Subject::isTetheringPossible() const noexcept
     {
-        return phoneMode != PhoneMode::Offline;
+        return phoneMode != PhoneMode::Offline && simSelected();
```

Comment:
please check if not nullptr befero calling simSelected.

Reply:
fixed